support-frontend/assets/pages/supporter-plus-landing/twoStepPages/secondStepCheckout.tsx

```diff
@@ -93,6 +96,17 @@ export function SupporterPlusCheckout({
 		</Button>
 	);
 
+	const tierPlanPeriod = contributionType.toLowerCase() as keyof TierPlans;
+	const billingPeriod = (tierPlanPeriod[0].toUpperCase() +
+		tierPlanPeriod.slice(1)) as BillingPeriod;
```

Comment:
Can we get this off `state.page.checkoutForm.product.billingPeriod`?

Reply:
Yep, done, had to ensure state was updated (it was'nt for some reason), commit here -> https://github.com/guardian/support-frontend/pull/5838/commits/1edea74d3f2eb4f41a4038de7354541de2e0ab49